Steg 2

Dyrker opened the pull request:
Mitt magnum opus

Dyrker committed:
Made checkboxes work

Making a commit before I try implementing removetask function

Dyrker committed:
AddTaskPage works

kingen löste det :)

Dyrker committed:
Filter funktion tillagd

Filterfunktion som använder provider är nu tillagd, kommentarer fattas, men commitar nu när allting funkar

Dyrker committed:
Namn på filet + PopupMenuButton istället för ShowDialog

Filernas namn ska nu va i enlighet med flutters regler samt bytte jag från showdialog till popupmenubutton för att det tydligen va en grej och funkar betydligt bättre

Dyrker committed:
Väldigt viktig uppdatering

Användaren får nu ett medelande när man markerar en task som Done

Dyrker committed:
Yttligare en mycket viktig uppdatering

När task listan e tom visas en fin gubbe som dansar samt en text, Appbar färgen är nu även satt till en fin färg

Dyrker committed:
formatering

Dyrker committed:
Kommentarer tillagda

rapunschel reviewed (commented):
Bra jobbat, från vad jag kan se så funkar allt som det ska med att lägga till nya todos och ta bort.
Synpunker:  
home_page_view
- När man lägger till en task sen bytar filter till "undone" så står det att listan är empty.
-  När man väljer ett filter, så är det otydligt (inget är markerat som valt),. Svårt att se nuvarande filter.

add_task_view
- 3 icon knapp som inte gör något.
- När man lägger till tasks så finns ingen tydlig feedback, tips är att antingen gå till todo view eller ha en snackbar som feedback.
- Saknas styling.

Säkert grejer du inte har hunnit fixa ännu.

Försök att undvika ha både kod och kommentarer på samma rad, ha kommentarer ovanför istället. Om kommentaren på en rad blir för lång så dela upp den i två. Underlättar när man läser.

Dyrker committed:
theme satt, Filterknapp visar nu aktivt filter

Numera sätts nästan allt via theme och appen har en ny snygg och lugnande look ;). Filterknappens icon är nu nuvarnde filtertext.

Dyrker commented:
Tack för feedback!
